packages/inference/src/lib/makeRequestOptions.ts

```diff
@@ -243,6 +243,10 @@ function makeUrl(params: {
 		}
 		default: {
 			const baseUrl = HF_HUB_INFERENCE_PROXY_TEMPLATE.replaceAll("{{PROVIDER}}", "hf-inference");
+			if (params.taskHint && ["feature-extraction", "sentence-similarity"].includes(params.taskHint)) {
+				/// when deployed on hf-inference, those two tasks are automatically compatible with one another.
+				return `${baseUrl}/pipeline/${params.taskHint}/${params.model}`;
```

Comment:
would be cleaner if it was `/{model}/pipeline/{task}`, probably (also simpler to route)